tests/unit/test_buildcmd_escaping.py

```diff
@@ -38,7 +38,7 @@ def setup_class(cls):
         cls.src_file_name = "main.cpp"
         cls.src_file_path = os.path.join(cls.tmp_dir,
                                          cls.src_file_name)
-        cls.compiler = "g++"
+        cls.compiler = "clang++"
```

Comment:
Why is this change introduced?

Reply:
I think this is a bug which is present since a while. `cls.compiler` is called with `--target` and gcc7  exits with an error (return value not equal with 0), however newer gcc like gcc 4.8 just gives a warning and returns with 0.